.github/workflows/promptflow-evals-e2e-test.yml

```diff
@@ -66,7 +66,6 @@ jobs:
           poetry run pip install -e ../promptflow-devkit
           poetry run pip install -e ../promptflow-tracing
           poetry run pip install -e ../promptflow-tools
-          poetry run pip install -e ../promptflow-azure
```

Comment:
Do we need same change for unit-test pipeline yaml?

Reply:
Changed.